### ISSUE TITLE
Add period and comma to letter keys on menu selection

### ISSUE THREAD

**Issue by andrewtavis** [scribe-org/Scribe-iOS#308]
### Terms

- [X] I have searched [open and closed feature requests](https://github.com/scribe-org/Scribe-iOS/issues?q=is%3Aissue+label%3Afeature)
- [X] I agree to follow Scribe-iOS' [Code of Conduct](https://github.com/scribe-org/Scribe-iOS/blob/main/.github/CODE_OF_CONDUCT.md)

### Description

This issue would be to implement the change to the keyboard that would be triggered by the menu item in #196. If the user chooses to include a period and comma in the letter keys via the menu, then a period would appear to the left of the spacebar and a comma to the right of the spacebar for letter keys, number keys and special keys.

- This issue is blocked by #196 

### Contribution

This issue will be done as a part of Google Summer of Code 2023 :) Happy to support where needed!

**Comment by SaurabhJamadagni** [comment]
Hey @andrewtavis! I just wanted to update you on the progress of this issue. I was able to figure out how to check if the user has the extensions installed. I also figured out establishing common data stores so different targets can save their preferences.

I haven't pushed the changes yet as it's a bit buggy right now and I'm trying to identify that problem. To be specific, currently it won't reload the list when a user adds or removes a language from Settings. Trying to figure out where would be a good place to refresh. 

I am sorry on the delay. But I'm a bit occupied with last minute shopping and packing for the move. I'll share the changes soon. Just wanted to inform about the delay 😅
